NavigationReactNative/src/NavigationBar.tsx

```diff
@@ -89,4 +89,4 @@ var Container: any = ({collapse, children, ...props}) => (
 var NVNavigationBar = requireNativeComponent<any>('NVNavigationBar', null);
 var NVToolbar = requireNativeComponent<any>('NVToolbar', null);
 
-export default Animated.createAnimatedComponent(NavigationBar);
+export default Animated.createAnimatedComponent(NavigationBar);
```

Comment:
Can you remove this change so this file doesn't show up in the diff, please? Looks like you've added a new line